src/Infrastructure/Mail/MailTemplateFilenameTraversable.php

```diff
@@ -14,7 +14,12 @@ public function __construct( private readonly string $mailTemplatePath ) {
 	}
 
 	public function getIterator(): \Iterator {
-		foreach ( glob( $this->mailTemplatePath . '/*\.twig' ) as $fileName ) {
+		$glob = glob( $this->mailTemplatePath . '/*\.twig' );
+		if ( $glob === false ) {
```

Comment:
@gbirke Should I re-throw the Exception as we are doing in the case of JsonException?

Reply:
Unlike the JSON functions, `glob` does not have a flag that changes its behavior from returning an error result vs throwing an exception: https://www.php.net/manual/en/function.glob
So we have to check for false here.